backend-api/template.yaml

```diff
@@ -1400,3 +1400,7 @@ Outputs:
   PrivateApiUrl:
     Description: Private API Gateway base URL
     Value: !Sub https://${PrivateApi}.execute-api.${AWS::Region}.amazonaws.com/${Environment}
+
+  StsMockApiUrl:
```

Comment:
If we know what the DNS is for each environment - do we need this, or could we assume it directly in the template?

Reply:
This was added so that the generate_env_file.sh can create an STS_MOCK_API_URL env var for testing main and custom stacks